src/Wrapper.js

```diff
@@ -11,9 +12,35 @@ import { getAppPath } from './routing'
 import { staticApps } from './static-apps'
 import { addressesEqual } from './web3-utils'
 import { noop } from './utils'
-import { APPS_STATUS_LOADING } from './symbols'
+import {
+  APPS_STATUS_ERROR,
+  APPS_STATUS_READY,
+  APPS_STATUS_LOADING,
+} from './symbols'
 
 class Wrapper extends React.Component {
+  static propTypes = {
+    account: PropTypes.string.isRequired,
+    apps: PropTypes.array.isRequired,
+    appsStatus: PropTypes.oneOf([
+      APPS_STATUS_ERROR,
+      APPS_STATUS_READY,
+      APPS_STATUS_LOADING,
+    ]).isRequired,
```

Comment:
Wonder if it'd be worth making a `prop-types.js` to export utilities like this, so we don't have to import constants / etc we don't use in some components?

Reply:
Off the top of my head: a `prop-types.js` to set commonly used `propTypes`, feels like it could get out of date very easily. Whereas having to import constants makes it explicit to the `types` the props need to follow.
Having said that, I don't have strong feelings towards either approach. Thoughts @bpierre?